library/Fission/CLI/Environment/Override.hs

```diff
@@ -83,7 +83,7 @@ write path env = writeBinaryFileDurable path $ YAML.encode env
 writeMerge :: MonadIO m => FilePath -> Override -> m ()
 writeMerge path newEnv = do
   currentEnv <- decode path
-  writeBinaryFileDurable path $ YAML.encode (currentEnv <> newEnv)
+  write path $ currentEnv <> newEnv
```

Comment:
Remind me what the difference is? Why don’t we want a durable write here?

Reply:
It's still a durable write, The diff doesn't show the full context: 
```haskell

-- | Writes partial environment to path
write :: MonadIO m => FilePath -> Override -> m ()
write path env = writeBinaryFileDurable path $ YAML.encode env

-- | Merge a partial env with an existing env at the path
writeMerge :: MonadIO m => FilePath -> Override -> m ()
writeMerge path newEnv = do
  currentEnv <- decode path
  write path $ currentEnv <> newEnv
```
we just had repeat logic

Reply:
Sweet 🍬